packages/extension/src/ui/features/accountTokens/AccountSubheader.tsx

```diff
@@ -72,6 +85,11 @@ export const AccountSubHeader: FC<AccountSubheaderProps> = ({
           </AccountStatusText>
         )}
       </div>
+      {sumCurrencyValue !== undefined && (
```

Comment:
```suggestion
      {sumCurrencyValue && (
```
as it will be a string there's no reason of 0 being false